packages/viewport-react/package.json

```diff
@@ -23,7 +23,7 @@
 		"@wordpress/compose": "^3.19.3"
 	},
 	"peerDependencies": {
-		"react": "^16.13.1",
+		"react": "^16.12.0",
```

Comment:
Why this downgrade?

Reply:
It's probably not necessary, but I was surprised how this seems to be the package the makes 16.13.1 be the version for the entire repo and went for consistency. Should be reverted.